tests/sentry/utils/test_csp.py

```diff
@@ -18,32 +18,3 @@
 def test_invalid_csp_report(report):
     with pytest.raises(InterfaceValidationError):
         Csp.to_python(report)
```

Comment:
I believe that we can ditch those, too. Validation is performed by Relay, and the only use of `to_python` is to create an in-memory model of the data. That is, unless it's completely dead code after removing that function from EventManager.

Reply:
Would like to keep until we remove the actual schema validation